www/tutorial/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "angular2-quickstart",
   "version": "1.0.0",
   "scripts": {
-    "start": "tsc && concurrently \"npm run tsc:w\" \"npm run lite\" ",
-    "lite": "lite-server",
-    "postinstall": "typings install",
-    "tsc": "tsc",
+    "build": "webpack",
+    "start": "webpack-dev-server  & open http://localhost:8080/",
+    "dev": "webpack-dev-server --inline --hot & open http://localhost:8080/",
+    "try-dev": "tsc && concurrently \"npm run tsc:w\" \"npm run lite\" ",
```

Comment:
I don't know why my webpack/dev script isn't rebuilding, so to unblock myself I did the try-dev script with concurrently and lite like it was before. would love to figure out why webpack isn't working, but just making sure it doesn't delay everything by a day :)